src/jvm/test/org/jetbrains/kotlinx/lincheck/test/representation/VerboseTraceTest.kt

```diff
@@ -24,10 +24,11 @@ import org.jetbrains.kotlinx.lincheck.*
 import org.jetbrains.kotlinx.lincheck.annotations.*
 import org.jetbrains.kotlinx.lincheck.strategy.managed.modelchecking.*
 import org.jetbrains.kotlinx.lincheck.test.*
+import org.jetbrains.kotlinx.lincheck.test.util.logVerbosePart
 import org.junit.*
 
 /**
- * This test checks `verboseTrace` option that makes Lincheck log all execution events.
+ * This test checks that makes Lincheck log all execution events in detailed part of output.
  */
 class VerboseTraceTest {
```

Comment:
Do we need this test?

Reply:
With out new approach to test - no. Removed.